platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/PlatformConstructor.java

```diff
@@ -105,7 +105,7 @@ static SocketFactory socketFactory(final KeysAndCerts keysAndCerts, final Crypto
         }
     }
 
-    static PlatformSigner platformSigner(final KeysAndCerts keysAndCerts) {
+    public static PlatformSigner platformSigner(final KeysAndCerts keysAndCerts) {
```

Comment:
Nullity annotations and checks on public methods.

Reply:
added